NOTES.md

```diff
@@ -1,3 +1,132 @@
+# Release 2.1.0-beta (Forbidden Planet)
+
+-- issues up-to-date with the 2.1 milestone as of 2015-07-12 20:00 UTC
+-- contributors " " next as of 3dac994 and docs:master as of 0b661c4
+
+This is a beta release for RethinkDB 2.1. **It is not for production use and has known bugs.** Please do not use this version for production data.
+
+We are looking forward to your bug reports on GitHub (http://github.com/rethinkdb/rethinkdb/issues/new) or on our mailing list (https://groups.google.com/forum/#!forum/rethinkdb).
+
+Release highlight:
+
+* Automatic failover
+* More flexible administration for servers and tables
+* Advanced recovery features
+
+Read the [blog post][2.1-beta] for more details.
+
+[2.1-beta]: http://rethinkdbblog.com/blog/2.1-beta/
+
+## Compatibility ##
+
+This beta release does not include automatic migration of data
+directories from older versions of RethinkDB. However the final 2.1.0
+version will include that feature for versions of RethinkDB down to
+1.14.0. Upgrading directly from RethinkDB 1.13 will require manually
+upgrading using `rethinkdb dump`.
+
+### Changed handling of server failures ###
+This release introduces a new system for dealing with server failures and network partitions.
+It is no longer necessary nor possible to permanently remove an unreachable server from the cluster through the system tables or the web UI. Instead the cluster now remains in an administrable state even with servers missing. To change the configuration of a given table, a majority of the servers that host replicas for that table need to be connected to guarantee data consistency. If more servers become unavailable, the new `emergency_repair` option to `table.reconfigure` can be used to restore table availability.
+
+This release adds full support for automatic failover. Tables with three or more replicas now automatically elect a new primary replica to restore availability after a currently configured primary replica has disconnected. No manual intervention is required in this scenario.
+
+### System table changes ###
+To reflect changes in the underlying cluster administration logic, some of the tables in the `"rethinkdb"` database changed.
+
+Each shard subdocument in the `"table_config"` table now has a new field `nonvoting_replicas`, that can be set to a subset of the servers in the `replicas` field. Furthermore `write_acks` must now be either `"single"` or `"majority"`. Custom write ack specifications are no longer supported. Instead non-voting replicas can be used to set up replicas that do not count towards the write ack requirements. Non-voting replicas can also be configured through the `table.reconfigure` command, by using the `novoting_replica_tags` optional argument. As a special case, `"table_config"` now lists tables that have all of their replicas disconnected as special documents with an `"error"` field.
```

Comment:
Is there a documentation page for how write acks work? It might be better to link to that, since this explanation is fairly long and weighty.